views/js/qtiCreator/editor/mathInput/mathInput.js

```diff
@@ -55,30 +55,98 @@ define([
+            pi:         { label: '&pi;',        latex: '\\pi',          fn: 'cmd',      desc: __('Pi') },
+            sin:        { label: 'sin',         latex: '\\sin',         fn: 'cmd',      desc: __('Sine') },
+            cos:        { label: 'cos',         latex: '\\cos',         fn: 'cmd',      desc: __('Cosine') },
+
+            // Operators
             times:      { label: '&times;',     latex: '\\times',       fn: 'cmd',      desc: __('Multiply') },
             divide:     { label: '&divide;',    latex: '\\div',         fn: 'cmd',      desc: __('Divide') },
-            plusminus:  { label: '&#177;',      latex: '\\pm',          fn: 'cmd',      desc:__( 'Plus/minus') }
+            plusminus:  { label: '&#177;',      latex: '\\pm',          fn: 'cmd',      desc: __('Plus/minus') },
+            equals:     { label: '=',           latex: '=',             fn: 'write',    desc: __('Equals') },
+
+            // Comparison
+            equiv:      { label: '≡',           latex: '\\equiv',       fn: 'cmd',      desc: __('Equivalent') },
+            sim:        { label: '~',           latex: '\\sim',         fn: 'cmd',      desc: __('Similar') },
+
+            // Geometry
+            parallel:   { label: '∥',           latex: '\\parallel',    fn: 'cmd',      desc: __('Parallel') },
+            perp:       { label: '⊥',           latex: '\\perp',        fn: 'cmd',      desc: __('Perpendicular') },
+
+            // Basic geometry
+            triangle:   { label: '△',           latex: '\\triangle',    fn: 'cmd',      desc: __('Triangle') },
+            angle:      { label: '∠',           latex: '\\angle',       fn: 'cmd',      desc: __('Angle') },
+
+            // Lines and vectors
+            segment: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo>&#x27F7;</mo></mover></math>',
+                latex: '\\overleftrightarrow{AB}',
+                fn: 'write',
+                desc: __('Line Segment')
+            },
+            vector: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo>&#x27F6;</mo></mover></math>',
+                latex: '\\overrightarrow{AB}',
+                fn: 'write',
+                desc: __('Vector')
+            },
+            ray: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo style="margin-bottom: 0.2rem;">&#x21c0;</mo></mover></math>',
+                latex: '\\stackrel{\\rightharpoonup}{AB}',
+                fn: 'write',
+                desc: __('Ray')
+            },
+            line: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo style="margin-bottom: 0.2rem;">&#x2E3A;</mo></mover></math>',
```

Comment:
I'm worried about accessibility, but I don't have anything to suggest so maybe we can ignore it for now and leave it for QA.

NVDA, for `overline` and `overparen`, doesn't announce anything at all, which is pretty bad.
For other three it announces `right arrow`, `left right arrow`, `right harpoon` which is more or less understandable. I guess it announces the "names" of unicode symbols.